packages/perseus/src/question-paragraph.tsx

```diff
@@ -19,7 +19,7 @@ class QuestionParagraph extends React.Component<Props> {
             <div
                 className={
                     this.props.inline
-                        ? this.props.className ?? undefined
+                        ? (this.props.className ?? undefined)
```

Comment:
I couldn't figure out why `prettier` changed its mind in this diff on this formatting. 🤷‍♂️